SdkTools.props

```diff
@@ -40,7 +40,7 @@
       <ScriptFilePath Condition=" '$(ScriptFilePath)' == '' ">$(RepositoryRootDirectory)\scripts\UpdateWintrust.ps1</ScriptFilePath>
     </PropertyGroup>
 
-    <Exec Command="$(PowerShellFilePath) -NonInteractive -ExecutionPolicy Unrestricted -Command &quot;&amp; { &amp;'$(ScriptFilePath)' '$(OutputPath)\tools\SDK\x64\wintrust.dll.ini' } &quot;" LogStandardErrorAsError="true" />
+    <Exec Command="$(PowerShellFilePath) -NonInteractive -ExecutionPolicy Unrestricted -Command &quot;&amp; { &amp;'$(ScriptFilePath)' '$(OutputPath)tools\SDK\x64\wintrust.dll.ini' } &quot;" LogStandardErrorAsError="true" />
```

Comment:
`$(OutputPath)` already has a trailing slash, so the output had an extra `\`.

`Adding NavSip.dll to wintrust.dll.ini - F:\git\sign\artifacts\bin\Sign.Cli\Debug\net8.0\win-x64\\tools\SDK\x64\wintrus
  t.dll.ini`